.eslintrc

```diff
@@ -2,7 +2,10 @@
     "parser": "babel-eslint",
```

Comment:
Where are these configs coming from? If the source is BRUI, can we import them instead?

Reply:
This is compiled from the .eslintrc files in Preview and Elements. The .eslintrc from BRUI is not available in the npm package so I cannot import it here. That could be something we do in the future so taht we can import the configs into Elements, Preview, Annotations, etc.